bear-requirements.txt

```diff
@@ -0,0 +1,42 @@
+HTTPolice~=0.5.2
+aenum~=2.0.8
+apertium-lint~=0.29
+autoflake~=0.7
+autopep8~=1.2
+bandit~=1.2,<1.7
+bashate~=0.5.1
+cmakelint~=1.3
+cppclean~=0.12.0
+cpplint~=1.3
+dennis~=0.9
+docutils-ast-writer~=0.1.2
+eradicate~=0.1.6
+git-url-parse~=1.1
+guess-language-spirit~=0.5.2
+html-linter~=0.4.0
+isort~=4.2
+language-check~=1.0
+libclang-py3~=3.4.0
+lxml>=1.0,<4.4.0
+memento-client~=0.6.1
+munkres~=1.1.2
+mypy==0.590
+nbformat~=4.1
+nltk~=3.2
+proselint~=0.7.0
+pycodestyle~=2.2
+pydocstyle~=2.0
+pyflakes~=2.0.0
+pylint~=1.7.2
+pyroma~=2.2.0
+pyyaml~=3.12
+radon==1.4.0
+restructuredtext-lint~=1.0
+rstcheck~=3.1
+safety~=1.8.2
+scspell3k~=2.0
+sqlparse~=0.2.4
+vim-vint~=0.3.12,!=0.3.19
+vulture~=0.25.0
+yamllint~=1.12.0
+yapf>=0.21.0<0.27.0
```

Comment:
This file does not belong here. Where is this even used in this repository?

Reply:
This doesn't seem right. I had cherry-picked the commits in the hope of making the builds green again. Yeah, it doesn't belong to this repo, and I'll make the necessary changes.